Userland/Utilities/less.cpp

```diff
@@ -540,6 +542,18 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     TRY(setup_tty(!dont_switch_buffer));
 
+    ScopeGuard teardown_guard([] {
+        teardown_tty();
+    });
+
+    auto teardown_sigaction_handler = [](auto) {
+        teardown_tty();
+        exit(1);
+    };
+    struct sigaction teardown_action;
+    teardown_action.sa_handler = teardown_sigaction_handler;
+    TRY(Core::System::sigaction(SIGTERM, &teardown_action, nullptr));
```

Comment:
why are we mixing `sigaction` and `signal`. surely one of them is the better choice and we should stick to it?

Reply:
I didn't initially replace the existing `signal()` calls with `Core::System::sigaction()`, as I didn't think it was strictly necessary. However, after a [quick search](https://stackoverflow.com/a/232711) it seems that using `sigaction` is preferred, unless there is a good reason not to do so.

I've now replaced calls to `Core::System::signal()` with `Core::System::sigaction()`.